### PR TITLE
Wallet v2 on boarding screens

### LINKED ISSUE #982: V2 Onboarding screens
Update godcr on boarding screens to the V2 layout designs.

https://www.figma.com/file/e0cYaAzkXyXXPY00v23cQY/GoDCR-%26-DCRMobile?node-id=1%3A2

image

### PR DESCRIPTION
**Pull request by Sirmorrison**
Fix #982 

image

* initial layout
image

image

image

image

* current layout
image

image

image

image

### CODE REVIEW

File: ui/decredmaterial/collapsible.go [hunk 1]
```diff
@@ -10,15 +10,34 @@ import (
 	"github.com/planetdecred/godcr/ui/values"
 )
 
+type IconStyle uint8
+
+const (
+	// Outline sets the outline material design chevron icon.
+	Chevron IconStyle = iota
+	// Fill sets the Fill chevron icon.
+	Caret
```
Comment (dreacot) [inline ui/decredmaterial/collapsible.go]: these comments might have been referring to other variables, and the comment doesn't match the variable name

**Review by dreacot — CHANGES_REQUESTED** [review]

File: ui/decredmaterial/collapsible.go (hunk 1 above)
Comment (Sirmorrison) [reply ui/decredmaterial/collapsible.go]: Nope... i renamed the data type because of conflict. let me update.

**Comment by dreacot** [comment]
https://github.com/planetdecred/godcr/issues/1003 doesn't seem to be solved

**Comment by Sirmorrison** [comment]
> #1003 doesn't seem to be solved

You wont be needing that approach to create an account after this PR is merged.

The wallet page will be fully deprecated after my next PR https://github.com/planetdecred/godcr/issues/985

**Review by dreacot — APPROVED** [review]

**Pull request merged**